run-dev-server.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+trap 'echo Stopping dev env...; kill $(jobs -p); exit' SIGINT SIGTERM
+
+export HLL_PASSWORD=<fill in yours>
+export HLL_PORT=<fill in yours>
+export HLL_HOST=<fill in yours>
+export HLL_DB_PASSWORD=<fill in yours>
+export DJANGO_DEBUG=True
+
+export DB_URL=postgres://rcon:$HLL_DB_PASSWORD@localhost:5432
+export REDIS_URL=redis://localhost:6379/0
+
+docker-compose -f docker-compose.yml -f docker-compose.dev.yml up -d redis postgres
+
+pip install -r requirements.txt
+PYTHONPATH=$PWD DJANGO_DEBUG=$DJANGO_DEBUG ./rconweb/manage.py runserver &
```

Comment:
you could add a `makemigration` and a `migrate` step to be totally hands off

Reply:
done (I guess... 🤦‍♂️ )